lib/paranoia.rb

```diff
@@ -12,6 +13,14 @@ def self.default_sentinel_value
     @@default_sentinel_value
   end
 
+  def self.default_recovery_window=(val)
+    @@default_recovery_window = val
+  end
+
+  def self.default_recovery_window
+    @@default_recovery_window
```

Comment:
Use thread-local variables to enable usage in threaded code (eg: within a sidekiq job):
```
    Thread.current[:default_recovery_window]
```